setup.py

```diff
@@ -26,13 +26,13 @@
 
 setuptools.setup(
     name='mws',
-    version='1.0.0dev11',
+    version='1.0.0dev12',
     maintainer="James Hiew",
     maintainer_email="james@hiew.net",
     url="http://github.com/jameshiew/mws",
     description=short_description,
     long_description=long_description,
-    packages=['mws'],
+    packages=['mws', 'mws.apis'],
```

Comment:
This is fixing a bug since we introduced the additional **folder apis**.
It is recommended to do it like this for simple projects, see for e.g. here https://setuptools.readthedocs.io/en/latest/python3.html, its also done with a point instead of a backslash.